apps/client/src/app/pipes/pipes/mob-name.pipe.ts

```diff
@@ -1,18 +1,22 @@
 import { Pipe, PipeTransform } from '@angular/core';
+
 import { LocalizedDataService } from '../../core/data/localized-data.service';
 
 @Pipe({
   name: 'mobName'
 })
 export class MobNamePipe implements PipeTransform {
-
+  public static GetActualMobId(id: number) {
```

Comment:
Why make this `public`? I can understand `static` for small optimization but not `public`. Also, in Typescript, method names are lowerCamelCase, not UpperCamelCase.

Reply:
I used it in `E:\ffxiv-teamcraft\apps\client\src\app\modules\list\data\ListRowSerializationHelper.ts`
and my bad on naming. I'm using primarily c# so I forget when I context switch.